src/helpers/shifts.helper.ts

```diff
@@ -6,8 +6,20 @@ import { ColorHelper } from "./colors/color.helper";
 import { Colors } from "./colors/color.model";
 import { VerboseDateHelper } from "./verbose-date.helper";
 import { VerboseDate } from "../common-models/month-info.model";
+import { DataRow } from "../logic/schedule-logic/data-row";
 
 export class ShiftHelper {
+  public static getWorkersCount(day: number, data: DataRow[]): number {
+    const matrix = data.map((d) => d.rowData(false, false));
```

Comment:
Szczegół, ale się przyczepię :D 
Moim zdaniem ty lepiej pasuje nazwa zmiennej `shifts` ponieważ osobiście u mnie `matrix` kojarzy się z macierzą w sensie matematycznym. A to przecież ona nie jest